docs/content/users/basics/commands.md

```diff
@@ -977,6 +977,19 @@ ddev restart my-project my-other-project
 ddev restart --all
 ```
 
+## `self-upgrade`
+
+Give instructions on how to update or upgrade DDEV. Although the command doesn't actually do the upgrade (since it must be handled differently on different systems) it tries to figure out what you need to do and explain it to you. It must be executed in the context of a project.
```

Comment:
Attempt to be more specific about the result (printed output), and say the same thing in fewer words:

```suggestion
Output instructions for updating or upgrading DDEV. The command doesn’t perform the upgrade, but tries to provide instructions relevant to your installation. Must be executed from the project context.
```